### PR TITLE
Blackduck issue induced by terraform-plugin-docs

### PR DESCRIPTION
**Pull request by taohe1012**
##### SUMMARY

terraform-plugin-docs has indirect dependency of yaml.v2, which induce blackduck security issue.
Had to comment out the terraform-plugin-docs to fix this.

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

#### Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note

```

#### Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

#### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

### CODE REVIEW

**Comment by rounak-adhikary** [comment]
Can we try removing terraform-plugin-docs dependency in this way? 
https://github.com/dell/terraform-provider-powerflex/pull/218

**Comment by taohe1012** [comment]
> Can we try removing terraform-plugin-docs dependency in this way? [dell/terraform-provider-powerflex#218](https://github.com/dell/terraform-provider-powerflex/pull/218)

Thanks @rounak-adhikary , code updated according to  [dell/terraform-provider-powerflex#218](https://github.com/dell/terraform-provider-powerflex/pull/218).

**Review by sapana05 — APPROVED** [review]

**Review by rounak-adhikary — APPROVED** [review]

File: docs/data-sources/storage.md [hunk 1]
```diff
@@ -219,7 +219,7 @@ Read-Only:
 - `dell_controller_battery` (Attributes) dell controller battery (see [below for nested schema](#nestedatt--storage--oem--dell--dell_controller_battery))
 
 <a id="nestedatt--storage--oem--dell--dell_controller"></a>
-### Nested Schema for `storage.oem.dell.dell_controller_battery`
+### Nested Schema for `storage.oem.dell.dell_controller`
 
 Read-Only:
```
Comment (forrestxia) [inline docs/data-sources/storage.md:225]: @taohe1012 Curious why there are so many doc updates? They were wrong and now we do the correction?

Comment (taohe1012) [reply docs/data-sources/storage.md:225]: @forrestxia Before the terraform-plugin-docs version is v0.16.0.
Now these two doc files are generated by terraform-plugin-docs@v0.19.4. And the new generated doc are correct.

**Review by forrestxia — APPROVED** [review]

**Pull request merged**